compiler/rustc_middle/src/mir/interpret/allocation.rs

```diff
@@ -399,10 +413,6 @@ impl<Prov: Provenance, Extra, Bytes: AllocBytes> Allocation<Prov, Extra, Bytes>
 
 /// Byte accessors.
 impl<Prov: Provenance, Extra, Bytes: AllocBytes> Allocation<Prov, Extra, Bytes> {
-    pub fn base_addr(&self) -> *const u8 {
-        self.bytes.as_ptr()
-    }
```

Comment:
This getter was added for extern-so mode but it's useless for that purpose since it doesn't allow mutating the memory.